extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -640,9 +652,20 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 	}
 
 
-	public async onPageEnter(): Promise<void> {
-
+	public async onPageEnter(pageChangeInfo: azdata.window.WizardPageChangeInfo): Promise<void> {
 		if (this.migrationStateModel.refreshDatabaseBackupPage) {
+			const isOfflineMigration = this.migrationStateModel._databaseBackup?.migrationMode === MigrationMode.OFFLINE;
+			const lastBackupFileColumnIndex = this._blobContainerTargetDatabaseNamesTable.columns.length - 1;
+			this._blobContainerTargetDatabaseNamesTable.columns[lastBackupFileColumnIndex].hidden = !isOfflineMigration;
+
+			this._networkShareButton.checked = false;
+			this._networkTableContainer.display = 'none';
+			this._networkShareContainer.updateCssStyles({ 'display': 'none' });
```

Comment:
is setting .display = 'none'  preferred over setting css styles to hide.

Reply:
I tried various combinations of using only .display = 'none' and using updateCssStyles, but for some reason, this setup was the only one that produced the expected behavior.